src/services/requestService.js

```diff
@@ -0,0 +1,23 @@
+import { Request, MultiRequest } from '../models';
+
+/**
+ * Class for managing trip request
+ */
+export default class RequestService {
+  /**
+   * @param {object} body - arrays of request object
+   * @returns {object} obej - return object
+   */
+  static async multiCityRequest({ body, user }) {
+    if (body.length < 1) throw new Error('The request cannot be empty');
```

Comment:
Instead of doing this ` if (body.length < 1) throw new Error('The request cannot be empty');`, please make use of our validation schema.